migrations/826-user-drop-source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users DROP COLUMN `source`;
```

Comment:
That sure shows us that we shouldn't remove the users with source != 0, thanks!

I'm not sure if it's a good idea to have a "dangling" column like that, with no idea what it's for, no link with the model... what do you think @kmaglione ?